### PR TITLE
Check for broken links automatically

### PR DESCRIPTION
**Pull request by saraedum**
I fixed some links that pointed nowhere.

Some links, such as the ones pointing to data packages can't work currently, since the data packages are not present in the output at all. I disabled them. Fixing them is #4 and #19.

### CODE REVIEW

File: docs/cv/.gitignore [hunk 1]
```diff
@@ -1,3 +0,0 @@
-/systems.md
```
Comment (saraedum) [inline docs/cv/.gitignore:1]: These files/directories are only created virtually while mkdocs is running. No need to ignore them (anymore.)

File: echemdb/make_pages.py [hunk 2]
```diff
@@ -106,23 +100,17 @@ def create_element_pages(elementname):
     templatemd['data']['element'] = elementname
     templatemd['title'] = 'echemdb - {} CV data'.format(elementname)
 
-    target = copy.deepcopy(TARGET_FOLDERS['elements']).replace('tobesubstituted', elementname)
```
Comment (saraedum) [inline echemdb/make_pages.py:109]: There is no need to `deepcopy` a string. A `replace` does not mutate strings but just returns a modified copy.

File: echemdb/make_pages.py [hunk 3]
```diff
@@ -106,23 +100,17 @@ def create_element_pages(elementname):
...
-    targetfile = TARGET_FOLDERS['path'] + target
-    os.makedirs(os.path.dirname(targetfile), exist_ok=True)
-    with open(targetfile, 'w') as f:
-        frontmatter.dump(templatemd, targetfile)
```
Comment (saraedum) [inline echemdb/make_pages.py:113]: No need to create the directories explicitly here. `mkdocs_gen_files.open` takes care of this for us, but only if we actually write to its `f` which we did not do before.

File: environment.yml [hunk 4]
```diff
@@ -4,19 +4,19 @@ channels:
...
+  - pip
   - pip:
+    - linkchecker
     - mkdocs-material
     - mkdocs-gen-files
     - markdown-include
     - mdutils
     - mkdocs-section-index
     - mkdocs-macros-plugin
-    - python-frontmatter
```
Comment (saraedum) [inline environment.yml:18]: this package was listed twice.

**Review by DunklesArchipel — APPROVED** [review]
looks good.

**Pull request merged**